### ISSUE TITLE
Pulumi web UI shows all items as complete, but task hasn't finished - some items aren't being shown

### ISSUE THREAD

**Issue by mikemaccana** [pulumi/pulumi#7335]
The Pulumi web UI marks all jobs as complete, with none outstanding, but the deploy hasn't finished:

![image](https://user-images.githubusercontent.com/172594/122473760-32517d00-cfba-11eb-91dc-e56e3ad617d0.png)

The command line UI however shows there are outstanding tasks - eg the autoscale group:

![image](https://user-images.githubusercontent.com/172594/122473664-14841800-cfba-11eb-9f7e-a1486141e8f1.png)

It seems the web UI is missing items (eg the autoscale group) that are shown in the command line UI.

**Comment by leezen** [comment]
Just to confirm since I'm doing a visual diff, you're referring to the CloudFormation stack resource shown as 'creating' in the CLI and not showing in the console UI?

**Comment by mikemaccana** [comment]
Yes! The CloudFormation stack resource isn't being shown in the web UI, which makes it looks like there's a problem in the web UI  (since everything appears done but the deployment hasn't finished). Thanks Lee.
